src/datasources/config-api/config-api.service.ts

```diff
@@ -38,10 +39,29 @@ export class ConfigApi implements IConfigApi {
 
   async getChain(chainId: string): Promise<Chain> {
     try {
-      const key = `${chainId}_chain`;
+      const cacheKey = `${chainId}_chain`;
       const field = '';
       const url = `${this.baseUri}/api/v1/chains/${chainId}`;
-      return await this.dataSource.get(key, field, url);
+      return await this.dataSource.get(cacheKey, field, url);
+    } catch (error) {
+      throw this.httpErrorFactory.from(error);
+    }
+  }
+
+  async getSafeApps(
+    chainId?: string,
+    clientUrl?: string,
+    url?: string,
+  ): Promise<SafeApp[]> {
+    try {
+      const cacheKey = `${chainId}_safe_apps`;
```

Comment:
Since `chainId` is optional what do you think about using this in the field parameters for the cache key. So the key is `safe_apps`. This is more in line with how we approach caching in the other methods.

Reply:
Yes, totally. Thanks for pointing it out 🙂

Reply:
Changed in [aba0246](https://github.com/5afe/safe-client-gateway-nest/commit/aba0246a22e5a542dacc772f2ff0f7673a8b1ac7)